notebooks/264-speculative-sampling/264-speculative-sampling.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #37.                q_cum = torch.cat((q_cum, q), dim=1)
</pre><p><code>q_cum</code> is not used anywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1410/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>